rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/configuration/security/Sw360AuthenticationProvider.java

```diff
@@ -23,15 +24,26 @@
 @Component
 public class Sw360AuthenticationProvider implements AuthenticationProvider {
 
+    @Value("${sw360.test-user-id}")
+    private String testUserId;
+
+    @Value("${sw360.test-user-password}")
+    private String testUserPassword;
+
+    // TODO Thomas Maier 15-12-2017
```

Comment:
Do you want to solve this TODO in this PR?

Reply:
This will be covered in #710